protostar/cheatable_starknet/callable_hint_locals/deploy_hint_local.py

```diff
@@ -50,5 +50,7 @@ def deploy_prepared(
     async def _run_deploy_prepared(self, prepared: PreparedContract):
         try:
             return await self._contracts_controller.deploy_prepared(prepared)
-        except (ContractsCheaterException, StarkException) as exc:
-            raise TransactionRevertException(str(exc), exc) from exc
+        except ContractsCheaterException as exc:
```

Comment:
I think this is too broad erro class to forward to cairo
```
class ConstructorInputTransformationException(ContractsCheaterException):
    pass

class ConstructorInvocationException(ContractsCheaterException):
    pass

```

Reply:
[Narrow down type of exception caught in hintlocal](https://github.com/software-mansion/protostar/pull/1792/commits/9fbe7cb116dee92453a34e9d6de5370c68ab47fe)